package.json

```diff
@@ -31,14 +31,13 @@
     "test": "jest"
   },
   "dependencies": {
-    "base62": "~0.1.1",
-    "commoner": "~0.8.4",
-    "esprima-fb": "1001.1001.1000-dev-harmony-fb",
-    "jstransform": "~1.0.0",
-    "recast": "~0.4.8"
+    "commoner": "^0.10.0",
+    "jstransform": "^9.1.0"
```

Comment:
Well, since you're doing so many changes already and updating dependencies... these should be moved down and we should get rid of the whole `bin` section in here. This module should not publish any executables.